airbyte-integrations/connectors/source-singlestore/src/testFixtures/java/io/airbyte/integrations/source/singlestore/AirbyteSingleStoreTestContainer.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.singlestore;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static java.util.Collections.singleton;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.Future;
+import org.apache.commons.lang3.StringUtils;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+public final class AirbyteSingleStoreTestContainer extends JdbcDatabaseContainer<AirbyteSingleStoreTestContainer> {
+  private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse(
+      "ghcr.io/singlestore-labs/singlestoredb-dev");
+  static final String DEFAULT_TAG = "latest";
+  private static final int PORT = 3306;
+  private static final int DEFAULT_STARTUP_TIMEOUT_SECONDS = 300;
+  private static final int DEFAULT_CONNECT_TIMEOUT_SECONDS = 120;
+
+  // Container defaults
+  static final String DEFAULT_DATABASE_NAME = "s2db";
+  static final String ROOT_USER = "root";
+  static final String DEFAULT_ROOT_USER_PASSWORD = "root";
```

Comment:
Can we somehow hide it in the GitHub secrets?

Reply:
Do we really need this? it's just a default password for test container. I moved to secrets only SINGLESTORE_LICENSE variable